src/flex-example-app-android.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.example.android.snake</id>
+    <filename>flex-example-app</filename>
+    <name>flex-example-app</name>
+    <versionNumber>1.0.0</versionNumber>
+    <extensions>
+        <extensionID>com.heyzap.sdk.extensions</extensionID>
+
+        <extensionID>com.heyzap.sdk.extensions.unityads</extensionID>
+        <extensionID>com.heyzap.sdk.extensions.facebook.audiencenetwork</extensionID>
+
+        <!--extensionID>com.pozirk.ads.AdMob</extensionID-->
```

Comment:
Removed it. I haven't tested this yet but will do it today. Actually looks that AdMob is working without this additional GPS ANE just with Vungle's one. I have installed Genymotion and will test with it today!